### ISSUE TITLE
Adding David Kennedy to DSVA Slack

### ISSUE THREAD

**Issue by anerbovig** [department-of-veterans-affairs/va.gov-team#40816]
Hello! Please add david.kennedy@adhocteam.us as a full member of DSVA Slack. David will be serving as the accessibility lead across the VAOS, 1010, Authenticated Experience, and shared services teams. Thank you!
